.github/workflows/apolo-release.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  push:
+    branches: [apolo]
+    tags: ["v*"]
+  pull_request:
+    branches: [apolo]
+
+jobs:
+  release:
```

Comment:
as we discussed, let's publish `:development` as a result of CD for `apolo` branch